http/file_server.ts

```diff
@@ -537,6 +542,14 @@ export async function serveDir(req: Request, opts: ServeDirOptions = {}) {
 
   if (!opts.quiet) serverLog(req, response!.status);
 
+  if (opts.headers) {
+    for (const header of opts.headers) {
+      const name = header.split(":")[0];
+      const value = header.split(":").slice(1).join(":");
```

Comment:
I think this should split only once.
```suggestion
      const headerSplit = header.split(":");
      const name = headerSplit[0];
      const value = headerSplit.slice(1).join(":");
```